### ISSUE TITLE
Work on Ligand BIC

### ISSUE THREAD

**Issue by stephaniewankowicz** [ExcitedStates/qfit-3.0#336]
BIC of qFit ligand should be on rotational bonds, not on # of atoms since we are not sampling every atom.

**Comment by blake-riley** [comment]
Note also that you've got _local_search() happening, which does a rigid-body search.
This samples translations and rotations.
As such, I think you should add 6 dof to account for this in the BIC $k$.

**Comment by jessicaflowers** [comment]
With the new ligand sampling method using RDKit, I ran several tests to evaluate what the optimal parameters are for BIC/if it is even necessary in the first place. I ran qFit with BIC on for five different conditions: k = nconfs, k = nconfs * natoms, k = nconfs * 1.5, k = nconfs * 0.75, and k = nconfs * 0.5 on our true positive dataset. I also ran with no BIC. I evaluated the output qFit models based on their RSCC and number of confs in the final model. I found that for RSCC, k = nconfs * natoms was the worst condition. All of the others, including BIC off, were nearly indistinguishable. For number of output conformers, I again found that k = nconfs * natoms was the worst condition. Across the true positive dataset, this condition only found a mean value of 1.8 confs per model. The other conditions were, again, very similar with a mean value of about 2.38. These results show that we can construct a BIC condition where we are not completely degrading the quality of the output models, however, running with BIC never improves the models compared to running without. This suggests that BIC is not necessary for qFit-ligand.
